components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -222,22 +229,100 @@ func (db *Postgres) GetServiceGroups(
 	return sgDisplay, nil
 }
 
-// ReleaseString returns the release string of the service group, our criteria are:
+// ReleaseString returns the release string of the service group.
+//
+// Example: From the package_ident:       'core/redis/0.1.0/20200101000000'
+//          The release string should be: '0.1.0/20200101000000'
+//
+// Criterias:
 // * if the group of services has only one release, it returns it.
 // * if it has several releases then it returns the string 'Several'
 // * if it has no releases, (which it is impossible but we should protect against it) it
 //   will return the string 'Unknown'
 func (sgh *serviceGroupHealth) ReleaseString() string {
-	switch rNumber := len(sgh.Releases); rNumber {
+	sgh.breakReleases()
+	switch rNumber := len(sgh.composedReleases); rNumber {
 	case 0:
 		return "Unknown"
 	case 1:
-		return sgh.Releases[0]
+		return sgh.composedReleases[0]
 	default:
 		return fmt.Sprintf("Several (%d)", rNumber)
 	}
 }
 
+// PackageString returns the package name string of the service group.
+//
+// Example: From the package_ident:            'core/redis/0.1.0/20200101000000'
+//          The package name string should be: 'core/redis'
+//
+// Criterias:
+// * if the group of services has only one package name, it returns it.
+// * if it has several package names then it returns the string 'Several'
+//   (this case could happen when two services are running a package from different origins)
+// * if it has no package name, (which it is impossible but we should protect against it) it
+//   will return the string 'Unknown'
+func (sgh *serviceGroupHealth) PackageString() string {
+	sgh.breakReleases()
+	switch pNumber := len(sgh.composedPackages); pNumber {
+	case 0:
+		return "Unknown"
+	case 1:
+		return sgh.composedPackages[0]
+	default:
+		return fmt.Sprintf("Several (%d)", pNumber)
+	}
+}
+
+// breakReleases is an internal function that is being called by PackageString() and
+// ReleaseString() to break down the multiple releases that a service group could have.
+// The function is designed to run only once for performance purposes, avoiding breaking
+// the releases down on each call.
+func (sgh *serviceGroupHealth) breakReleases() {
```

Comment:
This is the function that is doing all the heavy lifting for splitting the releases. I added a number of unit tests to cover all scenarios.